jupyterhub/jupyterhub/overlays/build/kustomization.yaml

```diff
@@ -4,9 +4,18 @@ bases:
 - ../../base
 resources:
 - jupyterhub-imagestream.yaml
+- jupyterhub-img-imagestream.yaml
 - jupyterhub-buildconfig.yaml
 - jupyterhub-dh-buildconfig.yaml
 
 commonLabels:
   opendatahub.io/component: "true"
   component.opendatahub.io/name: jupyterhub
+
+patchesJson6902:
+  - patch: jupyterhub-dc.yaml
```

Comment:
I think this should be `path:` not `patch`

See https://github.com/opendatahub-io/odh-manifests/pull/429#issuecomment-856411926

Reply:
Oh sorry, it's fixed right now, thanks!